test/e2e/azure_lb.go

```diff
@@ -224,7 +231,7 @@ func AzureLBSpec(ctx context.Context, inputGetter func() AzureLBSpecInput) {
 	WaitForServiceAvailable(context.TODO(), elbSvcInput, e2eConfig.GetIntervals(specName, "wait-service")...)
 
 	By("connecting to the external LB service from a curl pod")
-	svc, err = servicesClient.Get("ingress-nginx-elb", metav1.GetOptions{})
+	svc, err := servicesClient.Get("ingress-nginx-elb", metav1.GetOptions{})
```

Comment:
@jsturtevant @aramase do we expect this test to pass for ipv6? it's currently failing

Reply:
yeah, this test is expected to pass for IPv6. The ELB should be reachable with curl - `curl http://[2603:1030:805:1::e]`

Reply:
does it need a special nginx image to pass from that pod? https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/646/files#diff-80e9637d0d1561369473466d508ec50eL92

Reply:
This could be failing because it is missing the brackets around the ip address?  Not sure what `i.IP` is returning (https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/646/files/b84d1bee7f90c4281d64ed89296653998e8f1e35#diff-80e9637d0d1561369473466d508ec50eR239)

Reply:
The default `nginx:1.18` image doesn't listen on IPv6. So it does require an image that can actually listen and respond on IPv6.

```
server {
    listen       80;
    server_name  localhost;
```

Reply:
Do you know of a good public image that has it I could use?